middleware/buffer/source/octet.cpp

```diff
@@ -98,8 +100,12 @@ namespace casual
                }
                catch( ...)
                {
-                  if( common::exception::capture().code() == common::code::xatmi::argument)
-                     return CASUAL_OCTET_INVALID_HANDLE;
+                  const auto error = common::exception::capture();
+
+                  if( error.code() == common::code::xatmi::argument)
+                     return CASUAL_OCTET_INVALID_HANDLE; 
+
+                  common::stream::write( std::cerr, "error: ", error, '\n');
 
```

Comment:
Shall this be logged to log file or go to std::cerr? (Or possibly both?)

Reply:
This was more a way to make the code/behaviour consistent with other C-buffers and the question is not just valid for this buffer type